packages/opentelemetry-types/src/context/propagation/Propagation.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SpanContext } from '../../trace/span_context';
+
+/** Defines a propagation interface. */
+export interface Propagation {
```

Comment:
Propagation sounds more like a namespace. The interface should probably be called `Propagator`.

Also, as discussed in https://github.com/open-telemetry/opentelemetry-js/pull/48#discussion_r296409415 there needs to be an additional abstraction than `SpanContext` if `SpanContext` represents the W3C `SpanContext`.

Reply:
Renamed to `Propagator `.